python/pyspark/sql/streaming.py

```diff
@@ -714,6 +714,13 @@ def parquet(self, path, mergeSchema=None, pathGlobFilter=None, recursiveFileLook
             If None is set, the value of the SQL config
             ``spark.sql.parquet.int96RebaseModeInRead`` is used by default.
 
+        Other Parameters
```

Comment:
I didn't get why you leave parameters here, for instance `mergeSchema` but removed it in other places?

Reply:
Good point, @MaxGekk . this might be a kinda confusing part.

I'd say the root cause is the parameter definitions of existing Python functions are not consistent.
For example, there is case where the `mergeSchema` is not explicitly defined but just included in `**options`, whereas there is case where the parameter name is explicitly indicated as `mergeSchema` even though it is the same `parquet` function.

```python
def parquet(self, *paths, **options):
```

```python
def parquet(self, path, mergeSchema=None, pathGlobFilter=None, recursiveFileLookup=None,
    datetimeRebaseMode=None, int96RebaseMode=None):
```

So, I basically addressed the Python docstring depending on the several rules below:
- If parameter is not sprcified but included `**options` or `kwargs**`, remove the description and replace to the `Other Parameters`.
- If parameter is specified with name, just leave it as is in the `Parameters`.

I fully understand this is confusing.
Do you think maybe it would be better to fix the inconsistency of function definitions in this PR as well?

also cc @HyukjinKwon

Reply:
Can you remove the parameters such as `mergeSchema` except `path`?

Reply:
Sounds good, let me address it. Thanks, @HyukjinKwon !